src/partials/about.html

```diff
@@ -0,0 +1,12 @@
+<div class="">
+  <h2 class="About__title">ABOUT</h2>
+
+  <div>
+    <p class="">
+      We are a minimalistic flower studio. We love people and flowers. Therefore, every day we give
+      people the beauty and joy creating stylish and modern masterpieces. Every bouquet we create is
+      unique as you are.
+    </p>
+  </div>
+  <img class="" src="" alt="kwiaty" />
```

Comment:
pliki img są już w naszym projekcie, podlinkuj odpowiednią ścieżkę, oraz zrób to na zasadzie picture do opdowiednich wymiarów mobile,table, desktop

Reply:
`<picture>
  <source>
  <source>
  <img>
</picture>`

Reply:
picture
  source
  source
  img
picture

Reply:
media="telefon" src = ...
media = tablet src ...
img = desktop i sciezka

Reply:
media="telefon" src = ...
media = tablet src ...
img = desktop i sciezka